src/main/java/io/kestra/plugin/dlt/package-info.java

```diff
@@ -0,0 +1,8 @@
+@PluginSubGroup(
+    title = "DLT",
+    description = "This sub-group of plugins contains tasks for using DLT.\n",
+    categories = PluginSubGroup.PluginCategory.TOOL
```

Comment:
It can be mainly used for the E from ETL but also kind of for the T so let's put it:

```suggestion
    categories = {PluginSubGroup.PluginCategory.INGESTION, PluginSubGroup.PluginCategory.TRANSFORMATION}``